src/common/components/DatePicker/DatePicker.tsx

```diff
@@ -7,7 +7,8 @@ import { Input } from '../Inputs'
 import CalendarIcon from 'common/assets/icons/ico-calendar.svg'
 import ChevronIcon from 'common/assets/icons/ico-chevron.svg'
 
-import './DatePicker.module.css'
+import 'react-datepicker/dist/react-datepicker.css'
+import './DatePicker.css'
```

Comment:
I think the file name should be `DatePicker.module.css` http://pastel.wiki/en/Developers/pastel-electron-wallet/coding-style-guide

Reply:
@CrazyOptimist  Can you please add to the codin-style-guide when will we use `SomeComponent.module.css` and when will we use `SomeComponent.css`? Thanks

Reply:
Added here:
http://pastel.wiki/en/Developers/pastel-electron-wallet/coding-style-guide
Thanks!